api/db/setup_local_vars.sh

```diff
@@ -1,4 +1,4 @@
 source setup_vars.sh
 export WORKBENCH_PASSWORD=${WORKBENCH_PASSWORD:=workbench!pwd}
-export DB_SERVER=${DB_SERVER:=localhost:3306}
```

Comment:
This file will eventually go away, right -- this is just an intermediate step that keeps everything working?